### DISCUSSION TITLE
if hexo supports cursor that  a link mapping or jump to another page internal

### DISCUSSION THREAD

**Discussion by kirkzhangtech** [hexojs/hexo#5469] [Question / Answer]
The description as subject

**Comment by 7zMonkey** [comment]
If I understand correctly, is there a function similar to this
> 如果我理解的没错的话，是不是类似于这样的功能

![image](https://user-images.githubusercontent.com/130129908/232523420-237adc33-2eb9-48db-aa55-1f69935dc7e7.png)

If yes, probably you need a plugin like this
> 如果是的话，可能你需要一个这样的插件 [hexo-auto-association](https://www.npmjs.com/package/hexo-auto-association)
